### ISSUE TITLE
Alert bug: Group select screen: can't send alert.

### ISSUE THREAD

**Issue by danlatorre** [thewisecity/declarehome-android#1]
waiting to test with new build

**Comment by danlatorre** [comment]
Okay, I thought this was about text size and HCI factors, link hot zone collision due to small type. But it's still happening in the past two builds Oct 27 & 28.

**Comment by danlatorre** [comment]
Here's a screenshot of the error message I get: 
https://www.dropbox.com/s/0yobc2ktt4owh4o/Screenshot_2015-10-27-19-59-11.png?dl=0

I thinks I don't have a group selected, but I do. Same even if select another, or all.

**Comment by dexterlohnes** [comment]
Please send more details

**Comment by danlatorre** [comment]
Hit post alert, select alert category, select group (any or all) and tapping next generates the error. Every time.

**Comment by danlatorre** [comment]
Okay just got it to work. Was just cycling through the 3 groups, trying to post an alert in each one. And 1 alert was able to get past that Next button on the groups select screen and allowed me to post successfully and correctly. But then I did the same steps again and now I can't and get the same error at the same place, on the groups select screen.

**Comment by danlatorre** [comment]
Is the analytics helpful this is, from the error message generated, is that giving you any log detail?

**Comment by dexterlohnes** [comment]
Nope. Please try to find a pattern if you can since I can't repro so far.
Cell vs WiFi? Speed? Have you already taken other actions? What were they?
What if you close the app out then reopen it?

**Comment by danlatorre** [comment]
Okay. Hm. Will try more methods. Visual designers are also installing it 
so I can ask them too.

**Comment by danlatorre** [comment]
OK the repeatable Ux bug is this:
When I tap on the checkbox it doesn't think I've selected a group. When tap on the grouphone name it checks the box and knows I've selected it.
